tchannel/serializer/raw.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import absolute_import
+
+from tchannel.schemes import RAW
+
+
+class RawSerializer(object):
+    def __init__(self):
```

Comment:
These don't have any state... please remove this class in favor of regular old functions